### ISSUE TITLE
Add term hyperphalangy of the 4th finger

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10521]
**Preferred term label:**
Hyperphalangy of the 4th finger

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
An accessory phalanx of the third (middle) finger that is arranged linearly with the other phalanges. Hyperphalangy results from an accessory ossification center at the metacarpophalangeal joint. https://www.orpha.net/en/disease/detail/295002

**Parent term (use hpo.jax.org/app)**
Finger hyperphalangy HP:0030367

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Chondrodysplasia with joint dislocations, gpapp type (omim614078), PMID22887726

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000865
- label: Hyperphalangy of the 4th finger
- parents: [Finger hyperphalangy (HP:0030367)](https://hpo.jax.org/app/browse/term/HP:0030367)
- pmid: 
- def: An accessory phalanx of the fourth (ring) finger that is arranged linearly with the other phalanges. Hyperphalangy results from an accessory ossification center at the metacarpophalangeal joint.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10521
- synonyms: Hyperphalangy of fourth finger(NONE)
